spring-cloud-kubernetes-client-autoconfig/src/test/java/org/springframework/cloud/kubernetes/client/example/App.java

```diff
@@ -16,26 +16,12 @@
 
 package org.springframework.cloud.kubernetes.client.example;
 
-import io.kubernetes.client.openapi.ApiClient;
-import okhttp3.OkHttpClient;
-
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
-
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 /**
  * @author wind57
  */
 @SpringBootApplication
 public class App {
 
-	@Bean
```

Comment:
Can we add this back and put the a different App in your test?

Reply:
Sure thing.  I thought about going that route, but I couldn't find anywhere that the Bean from this class was used.